contracts/Fitup.sol

```diff
@@ -45,4 +53,9 @@ contract Fitup {
         }
         emit BetPayout(_issuer, _success, _organisation, _amount);
     }
+
+    function addNgo(string memory name, address organisation) external onlyOwner {
+        ngos.push(Ngo(name, organisation));
+        emit NgoAdded(name, organisation);
+    }
```

Comment:
I realized there is a problem in our design! How can we eliminate functions that might increase computation power? Also, if we add an ngo to NGO[] how we can delete it later?

Reply:
I am not sure i understand the eliminate functions to save computational power part can you explain a bit more? 

For the second part, i also though we should have a updatengo and a deletengo function. 

Do you agree? Should we have a call?